main/hunspell/APKBUILD

```diff
@@ -1,13 +1,13 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=hunspell
 pkgver=1.3.4
-pkgrel=0
+pkgrel=1
 pkgdesc="Spell checker and morphological analyzer library and program"
 url="http://hunspell.github.io/"
 arch="all"
 license="GPL2+ LGPL2+ MPL 1.1"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-depends=""
+depends="ncurses readline"
```

Comment:
This doesn’t work. You want to compile hunspell against ncurses and readline, i.e. you need their header files. These are included in *-dev packages; ncurses-dev and readline-dev in this case. The *-dev packages are needed only when making the package, so they should be declared in `makedepends`. Since abuild detects dynamically linked dependencies automatically, you don’t need to specify ncurses and readline in `depends`.

```diff
- makedepends="gettext-dev"
+ makedepends="gettext-dev ncurses-dev readline-dev"
```